mbq/client/contrib/permissions.py

```diff
@@ -164,6 +181,9 @@ def __init__(
             self.cache = None
             self.cache_period_seconds = None
 
+    def register_hook(self, name: str, function: Callable[..., Any]) -> None:
+        self.registrar.register(name, function)
```

Comment:
No need for this, can just passthrough to the registrar directly, e.g. `permissions_client.registar.register`